waspc/data/Generator/templates/react-app/src/auth/buttons/Generic.js

```diff
@@ -0,0 +1,39 @@
+{{={= =}=}}
+
+import config from '../../config.js'
+
+export const signInUrl = `${config.apiUrl}{= signInPath =}`
+export const logoUrl = '/images/{= iconName =}'
+
+const containerStyle = {
+  height: 40,
+  width: 225,
+  border: '1px solid darkgray',
+  borderRadius: 5,
+  padding: 5,
+  margin: '5px 0px',
+  backgroundColor: 'white'
+}
+
+const linkStyle = {
+  textDecoration: 'none',
+  color: 'black'
+}
+
+const logoStyle = {
+  maxWidth: '100%',
+  maxHeight: '100%',
+  display: 'inline-block',
+  marginRight: 10
+}
```

Comment:
This is where you can help some if you so choose, @matijaSos :D Thanks!

Reply:
nice! I'll play around with this and try to put it in Waspello, will propose some changes if I'll have ideas then.

Reply:
Sounds good, thanks! Merged to main, so if you do have any we can make it an update PR. Appreciate it! 👍🏻